app/models/admin/collection.rb

```diff
@@ -277,6 +280,17 @@ def default_lending_period
     self._default_lending_period || ActiveSupport::Duration.parse(Settings.controlled_digital_lending.default_lending_period).to_i
   end
 
+  alias_method :_cdl_enabled, :cdl_enabled
+  def cdl_enabled
```

Comment:
What do you think about renaming this `cdl_enabled?`?   Then it would match the method name in MediaObject, avoid the alias_method, and simplify the helper.